src/qt/guiutil_font.cpp

```diff
@@ -42,25 +42,18 @@ static std::map<FontFamily, std::vector<QFont::Weight>> mapSupportedWeights;
 
 FontFamily fontFamilyFromString(const QString& strFamily)
 {
-    if (strFamily == "SystemDefault") {
-        return FontFamily::SystemDefault;
-    }
-    if (strFamily == "Montserrat") {
-        return FontFamily::Montserrat;
+    for (const auto& [family, family_str] : AVAILABLE_FONTS) {
+        if (strFamily == family_str) {
+            return family;
+        }
     }
     throw std::invalid_argument(strprintf("Invalid font-family: %s", strFamily.toStdString()));
 }
 
 QString fontFamilyToString(FontFamily family)
 {
-    switch (family) {
-    case FontFamily::SystemDefault:
-        return "SystemDefault";
-    case FontFamily::Montserrat:
-        return "Montserrat";
-    default:
-        assert(false);
-    }
+    assert(family <= FontFamily::MaxVal);
```

Comment:
`family` is int -> it maybe less than 0

In this case `return AVAILABLE_FONTS[-1].second;` will make UB.

`assert(family >= 0 && family <= FontFamily::MaxVal);`

Reply:
Switched to using `uint8_t`, resolved in latest push